pipeline/src/test_util.rs

```diff
@@ -44,14 +44,12 @@ pub fn verify_asm_string<S: serde::Serialize + Send + Sync + 'static>(
         pipeline = pipeline.add_data_vec(&data);
     }
 
-    verify_pipeline(pipeline, BackendType::EStarkDump).unwrap();
+    verify_pipeline(pipeline).unwrap();
 }
 
-pub fn verify_pipeline(
-    pipeline: Pipeline<GoldilocksField>,
-    backend: BackendType,
-) -> Result<(), String> {
-    let mut pipeline = pipeline.with_backend(backend, None);
+pub fn verify_pipeline(pipeline: Pipeline<GoldilocksField>) -> Result<(), String> {
+    // TODO: Also test EStarkDumpComposite
+    let mut pipeline = pipeline.with_backend(BackendType::EStarkDump, None);
```

Comment:
I think the `verify()` function below *only* works with the `EStarkDump`, doesn't it? In practice, it was only called with that backend. So I removed the parameter and hard-coded it here.